### PR TITLE
Multi repos support

### PR DESCRIPTION
**Pull request by taizo**
Hello.

Some might want to update GitHub Secrets in multiple repositories by specifying two or more repos in `OWNER_REPOSITORY`.  eg. `OWNER_REPOSITORY: ${{ github.repository }},ScatteredTuna/supreme-tuna`.

@kneemaa  Could you take a look at the PR if you have time?

Thanks.

### CODE REVIEW

**Comment by kneemaa** [comment]
@taizo i just approved https://github.com/kneemaa/github-action-rotate-aws-secrets/pull/2 - do you mind resolving the conflicts and testing before I review?

**Comment by taizo** [comment]
@kneemaa Thank you for the reply.
Resolved the conflicts and confirmed that's working correctly. Could you please review the PR? 🙇‍♂️

**Comment by kneemaa** [comment]
To ensure I'm on the same page. The idea is that a single action could update the IAM keys of multiple repositories if the Github Dev Token has permissions?

File: rotate_keys.py [hunk 1]
```diff
@@ -133,6 +134,7 @@ def upload_secret(owner_repo,key_name,encrypted_value,pub_key_id,github_token):
     if updated_secret.status_code not in good_status_codes:
         print(f'Got status code: {updated_secret.status_code} on updating {key_name}')
```
Comment (kneemaa) [inline rotate_keys.py:135]: May want to let the user know which `owner_repo` failed to update.

Comment (taizo) [reply rotate_keys.py]: Agree 👍 I've just fixed.

**Comment by taizo** [comment]
> update the example on line 59 of README.md

Thank you for your suggestion.
I've updated the param usage part to add an example of multiple repos support instead of updating the example of line 59 because a user mostly want to update secret in single repository. The example of line 59 should be more general.

**Pull request merged**